addons/stock/models/stock_picking.py

```diff
@@ -612,28 +621,31 @@ def action_done(self):
 
     do_transfer = action_done #TODO:replace later
 
+    def _check_move_lines_map_quant_package(self, package):
```

Comment:
a docstirng would be nice to explain wha'ts this method doing.
also explain in the commit message that you only extracted this part of the method